rosidl_parser/rosidl_parser/__init__.py

```diff
@@ -445,8 +445,7 @@ def parse_value_string(type_, value_string):
     if type_.is_primitive_type() and not type_.is_array:
         return parse_primitive_value_string(type_, value_string)
 
-    # TODO(mikaelarguedas) change this condition once string escape function is implemented
-    if type_.is_primitive_type() and type_.is_array and type_.type != 'string':
+    if type_.is_primitive_type() and type_.is_array:
```

Comment:
Actually we do know what the default values should be in that case. Since the individual items are not quoted the default value contains an array with four elements. Otherwise they would need to be quoted.